code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -124,7 +124,7 @@
 	if(!istype(T))
 		return
 	if(reac_volume >= 5)
-		T.MakeSlippery(min_wet_time = 10, wet_time_to_add = reac_volume * 1.5)
+		T.MakeSlippery(TURF_WET_LUBE, min_wet_time = 10, wet_time_to_add = reac_volume * 1.5)
```

Comment:
Made this TURF_WET_LUBE since walking on cooking oil is fucking hard even with boots.